packages/terra-overlay/tests/jest/intl-context-setup.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable react/jsx-boolean-value, import/no-extraneous-dependencies */
+import { IntlProvider, intlShape } from 'react-intl';
```

Comment:
You can import these from terra-base now which exports them from react-intl. This helps us better match which version of react-intl is used.